src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -173,36 +173,43 @@ To format one or more lines of code, insert three backticks ( ```` ``` ```` ) ab
   </Collapser>
 </CollapserGroup>
 
-## Customer-specific values [#value-placeholders]
+## Highlighting lines of code [#highlight-code]
```

Comment:
@zuluecho9 somewhere in this section, you might insert a callout that explains that we can't highlight anything less that full lines--no highlighting of individual words or groups or words.